drivers/wifi/winc1500/bsp/include/nm_bsp.h

```diff
@@ -0,0 +1,284 @@
+/**
+ *
+ * \file
+ *
+ * \brief WINC BSP API Declarations.
+ *
+ * Copyright (c) 2016 Atmel Corporation. All rights reserved.
```

Comment:
I left the function prototypes in this file and moved the other definitions in ext